cookbooks/bcpc/recipes/cobbler.rb

```diff
@@ -345,3 +350,13 @@
   action [:enable,:start]
   notifies :run, 'bash[cobbler-sync]', :delayed
 end
+
+#
+# After recurring problems with reloading the xinetd
+# configuration, we resort to a forced restart on every chef run.
```

Comment:
I still don't think this commit has fixed my problem.  xinetd fails to respond on the TFTP port when the VMs boot, and I don't know why.  Worse, it works again after another chef run.